src/de/uni_passau/fim/gitwrapper/EventDataProcessor.java

```diff
@@ -43,7 +45,18 @@ class EventDataProcessor implements JsonDeserializer<EventData>, JsonSerializer<
         map.put("unlabeled", EventData.LabeledEventData.class);
         map.put("referenced", EventData.ReferencedEventData.class);
         map.put("merged", EventData.ReferencedEventData.class);
-        map.put("closed", EventData.ReferencedEventData.class);
+        map.put("closed", EventData.StateChangedEventData.class);
+        map.put("connected", EventData.ConnectedEventData.class);
+        map.put("reopened", EventData.StateChangedEventData.class);
```

Comment:
Question 1: Which effect does it have that "closed" now uses the `StateChangedEventData` instead of `ReferencedEvent` data? Just want to make sure that we don't lose information when getting the new resolution type now.

Question 2: Why has "reopened" been added?  I assume because of the resolution type stuff. But how does "reopened" affect the resolution?

Request: Could you please sort the events in a way that closed and reopened are next to each other (and don't have "connected" in between).


Could you please figure that out?